### ISSUE TITLE
Pydantic computed field example values

### ISSUE THREAD

**Issue by rsaleev** [pydantic/pydantic#7864]
### Initial Checks

- [X] I have searched Google & GitHub for similar requests and couldn't find anything
- [X] I have read and followed [the docs](https://docs.pydantic.dev) and still think this feature is missing

### Description

Maybe my approach is not right but I use `computed_field` wrapper for injecting ORM values in root model and it will be for sure a cool feature to have examples like for `Field`, less code and integration with FastAPI/OpenAPI 

### Affected Components

- [ ] [Compatibility between releases](https://docs.pydantic.dev/changelog/)
- [ ] [Data validation/parsing](https://docs.pydantic.dev/concepts/models/#basic-model-usage)
- [ ] [Data serialization](https://docs.pydantic.dev/concepts/serialization/) - `.model_dump()` and `.model_dump_json()`
- [X] [JSON Schema](https://docs.pydantic.dev/concepts/json_schema/)
- [ ] [Dataclasses](https://docs.pydantic.dev/concepts/dataclasses/)
- [ ] [Model Config](https://docs.pydantic.dev/concepts/config/)
- [X] [Field Types](https://docs.pydantic.dev/api/types/) - adding or changing a particular data type
- [ ] [Function validation decorator](https://docs.pydantic.dev/concepts/validation_decorator/)
- [ ] [Generic Models](https://docs.pydantic.dev/concepts/models/#generic-models)
- [ ] [Other Model behaviour](https://docs.pydantic.dev/concepts/models/) - `model_construct()`, pickling, private attributes, ORM mode
- [ ] [Plugins](https://docs.pydantic.dev/) and integration with other tools - mypy, FastAPI, python-devtools, Hypothesis, VS Code, PyCharm, etc.

**Comment by sydney-runkle** [comment]
Hi @rsaleev,

Thanks for your feedback. Could you provide an example MRE code snippet so that we can better understand what you're asking for? Thanks 👍

**Comment by rsaleev** [comment]
> Hi @rsaleev,
> 
> Thanks for your feedback. Could you provide an example MRE code snippet so that we can better understand what you're asking for? Thanks 👍

```
computed_field(alias="marketPlaceAccountName", return_type=str)
 def marketplace_acccount_name(self):
        return self.marketplace.name
``` 
No examples available for OpenAPI, only attr will be shown with field type

```
 order_number: Optional[str] = Field(
        serialization_alias="orderNumber", examples=[None, "1234567"]
    )
```
Examples will be shown in rendered OpenAPI docs

**Comment by sydney-runkle** [comment]
@rsaleev,

Thanks for the response. This seems like a reasonable feature request - I think we could also include `json_schema_extra` and a few other attributes from `Field` for `computed_field`.

**Comment by TiansuYu** [comment]
Started looking into this issue without hard commitment (Also leave the comment as a marker, so I can find the page again).

**Comment by alexmojaki** [comment]
PR: https://github.com/pydantic/pydantic/pull/8013

**Comment by adriangb** [comment]
I see we merged #8013 but I do want to point out that we should also make `Annotated` and `Field` in the return type work:

```python
from typing import Annotated

from pydantic import BaseModel, Field, computed_field


class Model(BaseModel):
    a: int
    b: int

    @computed_field
    def c(self) -> Annotated[int, Field(description='The sum of a and b', examples=[3, 5])]:
        return self.a + self.b

print(Model.model_json_schema())
```

Then we don't _need_ to re-add all of the arguments to `computed_field`.
